src/app/pages/MediaArticlePage/MediaArticlePage.styles.ts

```diff
@@ -86,4 +91,13 @@ export default {
         padding: `${spacings.DOUBLE}rem`,
       },
     }),
+
+  responsiveComponentWrapper: ({ mq }: Theme) =>
+    css({
+      marginBottom: `${GEL_SPACING_TRPL}`,
+      [mq.GROUP_4_MIN_WIDTH]: {
+        marginBottom: `${GEL_SPACING}`,
+        padding: `${GEL_SPACING_DBL}`,
+      },
```

Comment:
And just to add to what Aaron has mentioned above, if you do use the spacings from the Theme, you need to append `rem` to the end 

e.g. 
```suggestion
  responsiveComponentWrapper: ({ mq, spacings }: Theme) =>
    css({
      marginBottom: `${spacings.TRIPLE}rem`,
      [mq.GROUP_4_MIN_WIDTH]: {
        marginBottom: `${spacings.FULL}rem`,
        padding: `${spacings.DOUBLE}rem`,
      },
```

Reply:
Done